tests/unit/type/list.tcl

```diff
@@ -355,6 +613,54 @@ start_server {
       assert_equal {foo} [r lrange list3 0 -1]
     }
 
+    test "Linked BRPOPRPUSH" {
+      set rd1 [redis_deferring_client]
+      set rd2 [redis_deferring_client]
+
+      r del list1 list2 list3
+
+      $rd1 brpoprpush list1 list2 0
+      $rd2 brpoprpush list2 list3 0
+
+      r rpush list1 foo
```

Comment:
in order to gain more value in these, let's add some pre-existing value in list3, so that we can check the element got pushed to the right place?
or alternatively, if we think we got that covered already, and this test is just to check that a series of linked operations work, let's trim the additional 3 tests. (and maybe use two different types of BxPOPxPUSH in the original one test).

Reply:
I removed the additional tests and used a different poppush as you suggested.